src/main/java/com/iota/iri/service/snapshot/impl/SnapshotServiceImpl.java

```diff
@@ -42,6 +42,21 @@
      */
     private static final Logger log = LoggerFactory.getLogger(SnapshotServiceImpl.class);
 
+    /**
+     * Holds a limit for the amount of milestones we go back in time when generating the solid entry points (to speed up
+     * the snapshot creation).<br />
+     * <br />
+     * Note: Since the snapshot creation is a "continuous" process where we build upon the information gathered during
+     *       the creation of previous snapshots, we do not need to analyze all previous milestones but can rely on
+     *       slowly gathering the missing information over time. While this might lead to a situation where the very
+     *       first snapshots taken by a node might generate snapshot files that can not reliably be used by other nodes
+     *       to sync it is still a reasonable trade-off to reduce the load on the nodes. We just assume that anybody who
+     *       wants to share his snapshots with the community as a way to bootstrap new nodes will run his snapshot
+     *       enabled node for a few hours before sharing his files (this is a problem in very rare edge cases when
+     *       having back-referencing transactions anyway).<br />
+     */
+    private static final int OUTER_SHELL_SIZE = 100;
```

Comment:
Maybe this should be a configuration?

Reply:
Actually whether it should be a configured value or not it is a product question so @jakubcech should be involved.

Reply:
I don't think this should be made configurable from the get-go. There's already quite a lot to configure and wrap your brain around with different intervals and limits, and this seems like a reasonable thing to have set to a default value and possibly revisit in the future if we see a need for configurability.